src/components/YourCompany/NameTranslations/CorrectNameTranslation.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <name-translation
+        :nameTranslations="getNameTranslations"
+        :originalNameTranslations="originalNameTranslations"
+        @nameTranslationsChange="setNameTranslations"
```

Comment:
In this case I would prefer to see
```
@nameTranslationsChange="setNameTranslations($event)"
```

@ Other Devs: If you strongly disagree with my preferred style for event handlers, let's discuss... I could change my mind 😀

Reply:
If that event is receiving multiple props from the child emitter, it makes sense to do it like `setNameTranslations` ( cuz i don't know a better way to handle multiples :P ), but if it is a single prop being emit i think i also prefer to see the `$event`. 
TLDR: No objects to either approach 😁

Reply:
@cameron-freshworks https://www.google.com/search?q=vue+emit+multiple+parameters